src/components/form.js

```diff
@@ -108,6 +108,9 @@
         };
 
         const trigger = (data, loading, error) => {
+          if (data || error) {
+            B.triggerEvent('onActionDone');
```

Comment:
We have `onActionSuccess` when there is data and `onActionError` when there is an error. Why should we need this extra event?

Reply:
It will save a lot of extra interactions. For instance, wether there's an error or success doesn't matter: you always wanna remove the loading state of the button when the submit is done.